Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -103,104 +105,111 @@
 from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
-trackingLowPU.toReplaceWith(globalreco_tracking, _globalreco_tracking_LowPU)
+trackingLowPU.toReplaceWith(globalreco_trackingTask, _globalreco_tracking_LowPUTask)
 ##########################################
 # offlineBeamSpot is reconstructed before mixing in fastSim
 ##########################################
-_fastSim_globalreco_tracking = globalreco_tracking.copyAndExclude([offlineBeamSpot,MeasurementTrackerEventPreSplitting,siPixelClusterShapeCachePreSplitting])
-fastSim.toReplaceWith(globalreco_tracking,_fastSim_globalreco_tracking)
-
-_phase2_timing_layer_globalreco_tracking = globalreco_tracking.copy()
-_phase2_timing_layer_globalreco_tracking += fastTimingGlobalReco
-phase2_timing_layer.toReplaceWith(globalreco_tracking,_phase2_timing_layer_globalreco_tracking)
-
-globalreco = cms.Sequence(globalreco_tracking*
-                          particleFlowCluster*
-                          ecalClusters*
-                          caloTowersRec*
-                          egammaGlobalReco*
-                          jetGlobalReco*
-                          muonGlobalReco*
-                          pfTrackingGlobalReco*
-                          muoncosmicreco*
-                          CastorFullReco)
-
-_run3_globalreco = globalreco.copyAndExclude([CastorFullReco])
-run3_common.toReplaceWith(globalreco, _run3_globalreco)
+_fastSim_globalreco_trackingTask = globalreco_trackingTask.copyAndExclude([offlineBeamSpot,MeasurementTrackerEventPreSplitting,siPixelClusterShapeCachePreSplitting])
+fastSim.toReplaceWith(globalreco_trackingTask,_fastSim_globalreco_trackingTask)
+
+_phase2_timing_layer_globalreco_trackingTask = globalreco_trackingTask.copy()
+_phase2_timing_layer_globalreco_trackingTask.add(fastTimingGlobalRecoTask)
+phase2_timing_layer.toReplaceWith(globalreco_trackingTask,_phase2_timing_layer_globalreco_trackingTask)
+
+globalrecoTask = cms.Task(globalreco_trackingTask,
+                          particleFlowClusterTask,
+                          ecalClustersTask,
+                          caloTowersRecTask,
+                          egammaGlobalRecoTask,
+                          jetGlobalRecoTask,
+                          muonGlobalRecoTask,
+                          pfTrackingGlobalRecoTask,
+                          muoncosmicrecoTask,
+                          CastorFullRecoTask)
+globalreco = cms.Sequence(globalrecoTask)
+
+_run3_globalrecoTask = globalrecoTask.copyAndExclude([CastorFullRecoTask])
+run3_common.toReplaceWith(globalrecoTask, _run3_globalrecoTask)
 
 _fastSim_globalreco = globalreco.copyAndExclude([CastorFullReco,muoncosmicreco])
 # insert the few tracking modules to be run after mixing back in the globalreco sequence
 _fastSim_globalreco.insert(0,newCombinedSeeds+trackExtrapolator+caloTowerForTrk+firstStepPrimaryVerticesUnsorted+ak4CaloJetsForTrk+initialStepTrackRefsForJets+firstStepPrimaryVertices)
```

Comment:
Errors are derived from "firstStepPrimaryVerticesUnsorted+ak4CaloJetsForTrk" in this line.
I will check this part  again.